Latest version of tconnectsync-heroku fails to deploy

**Describe the problem**
I'm attempting to deploy the latest version of tconnectsync-heroku (2.1.0) to upgrade to Tandem Source compatibility, but I'm getting a failure when deploying: `ImportError: cannot import name 'TandemSourceChooseDevice' from 'tconnectsync.sync.tandemsource.choose_device'`

**To Reproduce**
1. Deploy the latest version of tconnectsync-heroku. 
    - **Note:** I am not deploying to Heroku, but to Dokku, a self-hosted version that supports Heroku-compatible applications using Heroku buildpacks. I have been deploying here the whole time I've used this project so I don't think the platform itself is related.

**Expected behavior**
- Deploying builds and runs successfully without errors.

**Have you followed the [Troubleshooting steps in the README?](https://github.com/jwoglom/tconnectsync-heroku#troubleshooting)** Yes

**Log output**
```
[2024-10-02 20:08:19 +0000] [13] [INFO] Starting gunicorn 23.0.0
[2024-10-02 20:08:19 +0000] [13] [INFO] Listening at: http://0.0.0.0:5000 (13)
[2024-10-02 20:08:19 +0000] [13] [INFO] Using worker: sync
[2024-10-02 20:08:19 +0000] [166] [INFO] Booting worker with pid: 166
The TCONNECTSYNC_HEROKU_SECRET environment variable is undefined,
so the API endpoints will not be accessible.
The TCONNECTSYNC_HEROKU_FEATURES environment variable is undefined,
so the following default tconnectsync features will be used: BASAL,BOLUS,PUMP_EVENTS,PROFILES
[2024-10-02 20:08:22 +0000] [166] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
    worker.init_process()
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 135, in init_process
    self.load_wsgi()
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
    return self.load_wsgiapp()
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/util.py", line 370, in import_app
    mod = importlib.import_module(module)
  File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/app/app.py", line 4, in <module>
    from utils import token_required, call, setup, get_time_args, run_update, as_text, parse_features
  File "/app/utils.py", line 13, in <module>
    from tconnectsync.sync.tandemsource.heroku_helpers import run_oneshot
  File "/app/.heroku/python/lib/python3.9/site-packages/tconnectsync/sync/tandemsource/heroku_helpers.py", line 2, in <module>
    from .choose_device import TandemSourceChooseDevice
ImportError: cannot import name 'TandemSourceChooseDevice' from 'tconnectsync.sync.tandemsource.choose_device' (/app/.heroku/python/lib/python3.9/site-packages/tconnectsync/sync/tandemsource/choose_device.py)
[2024-10-02 20:08:22 +0000] [166] [INFO] Worker exiting (pid: 166)
[2024-10-02 20:08:22 +0000] [13] [ERROR] Worker (pid:166) exited with code 3
[2024-10-02 20:08:22 +0000] [13] [ERROR] Shutting down: Master
[2024-10-02 20:08:22 +0000] [13] [ERROR] Reason: Worker failed to boot.
```

--- a/Pipfile
+++ b/Pipfile
@@ -7,7 +7,7 @@ verify_ssl = true
 
 [packages]
 flask = "*"
-tconnectsync = ">=2.1.0"
+tconnectsync = ">=2.1.3"
 flask-apscheduler = {git = "https://github.com/viniciuschiele/flask-apscheduler"}
 gunicorn = "*"
 

